src/Serilog/Context/LogContext.cs

```diff
@@ -12,9 +12,16 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+
 #if LOGCONTEXT
```

Comment:
I'm not entirely clear on what 5.1 is. I thought from [standard platform](https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/standard-platform.md) that it was 4.5, but if I try to compile with either `AsyncLocal` or `CallContext` it fails on 5.1.